src/IoT.Device.Graphics/DictionaryCharByte.cs

```diff
@@ -97,5 +97,26 @@ public bool ContainsKey(char cr)
 
             return false;
         }
+
```

Comment:
Commented code should be removed if not needed or used ;)

Reply:
Forgot about this one. Thank you!